Assets/RTLTMPro/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "com.engage.rtltmpro",
   "displayName": "RTL Text Mesh Pro",
-  "version": "3.4.6-engage.2",
+  "version": "3.4.7-engage",
```

Comment:
This should be `3.4.6-engage.3` in my opinion

Reply:
Yes, I need to put it back. UMP supports semantic versioning, but I didn't see our -engage.2 showing up when it was set that way before.